docs/resources/sql_global_config.md

```diff
@@ -17,13 +19,32 @@ resource "databricks_sql_global_config" "this" {
 }
 ```
 
+### Azure example
+
+For Azure you should use the `data_access_config` to provide the service principal configuration. You can use the Databricks SQL Admin Console UI to help you generate the right configuration values.
+
+```hcl
+resource "databricks_sql_global_config" "this" {
+  security_policy      = "DATA_ACCESS_CONTROL"
+  data_access_config = {
+    "spark.sql.session.timeZone" : "UTC",
+    "spark.hadoop.fs.azure.account.auth.type.lakehouse.dfs.core.windows.net" : "OAuth",
```

Comment:
can you remove `.lakehouse.dfs.core.windows.net` suffix from keys here?
because probably you've filled storage, but `Storage account of the ADLS instance the service principal has access to, not to be confused with container name. If blank then this service principal will be used as a fallback for all ADLS accesses.`